pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -56,6 +56,23 @@
             System.ComponentModel.Composition;
         </Value>
     </ValidatePackageSuppression>
+
+    <ValidatePackageSuppression Include="PermitInbox">
```

Comment:
Why do we require a suppression here?  Is it because the index isn't being updated?  If so, please make sure to file an issue and add a comment.